### DISCUSSION TITLE
Charts Update Is Delayed

### DISCUSSION THREAD

**Discussion by r01k** [locustio/locust#2769] [General]
On Locust 2.29.0 (Python 3.12.4) the charts update with an increasing lag behind the user activity. After about 20 minutes in the test the charts have already fallen behind by about 10 minutes. Is anyone else seeing this?

![image](https://github.com/locustio/locust/assets/19595786/60b15643-ecc3-48e5-8fa5-99b210b1befc)

**Comment by cyberw** [comment]
Sounds like a bug, so I created an issue (2771). Lets continue discussion there.
